docassemble/AssemblyLine/data/questions/al_visual.yml

```diff
@@ -70,14 +66,22 @@ initial: True
 code: |
   # Set the dropdown/hamburger menu items
   # by default we just add a "Start over" link
-  if _internal.get('steps') > 1:
+  if _internal.get('steps') >= 1:
```

Comment:
There's probably some compilated-ness to how 'steps' are handled, but everytime I start an interview through the playground, this shows "Start over" on the first page. If that's intentional, we don't need this if statement.

Reply:
Good point. >1 seems to only show "Start over" on screen 3. but >= 1 shows it on the first screen. very confusing. "Start over" is probably not needed on the first screen though so I can look at this more closely.